screens/questions/question.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="stylesheet" href="https://rocket-ui-library.netlify.app/stylesheets/components.css">
+    <link rel="stylesheet" href="https://rocket-ui-library.netlify.app/landing.css">
+    <link rel="stylesheet" href="/style.css">
+    <link rel="stylesheet" href="./question.css">
```

Comment:
Import all the CSS files to one main file and use that across.

Reply:
Ok,thanks